File: framework/languages/java/e2e/AutomaticTurnDispatch/Shared/src/main/java/systems/zlink/e2e/automaticturn/shared/Env.java
package systems.zlink.e2e.automaticturn.shared;

public final class Env {
    private Env() {
    }

    public static String get(String name) {
        return get(name, "");
    }

    public static String get(String name, String fallback) {
        String value = System.getenv(name);
        if (value == null || value.isBlank()) {
            return fallback;
        }
        return value;
    }
}

File: framework/languages/java/e2e-kotlin/RuntimeMonitoring/Shared/src/main/kotlin/systems/zlink/e2e/kotlin/runtimemonitoring/Env.kt
package systems.zlink.e2e.kotlin.runtimemonitoring

object Env {
    @JvmStatic
    @JvmOverloads
    fun get(
        name: String,
        fallback: String = "",
    ): String {
        val value = System.getenv(name)
        if (value.isNullOrBlank()) {
            return fallback
        }
        return value
    }
}
